### DISCUSSION TITLE
I enter the game and it gets stuck on a black screen.

### DISCUSSION THREAD

**Discussion by Krisstobal** [thatskymod/Sky-CotL-Scripts#119] [Q&A] [answered]
I enter canvas, go to play and it gets stuck on a black screen that goes on forever, it doesn't happen in the original game and I have already restarted my phone once and re-installed canvas too, please help I really need candles for the new season :( Is it because of the new update? It has been like this for at least 8 hours now.
 Thanks beforehand.

**Comment by vadallia** [comment]
yeah having the same problem

**Comment by TeasyDeer** [comment]
Same! But instead of a black screen, I get kicked out. I deleted and installed canvas seven times and restarted my phone about two.

**Comment by Krisstobal** [comment]
> I enter canvas, go to play and it gets stuck on a black screen that goes on forever, it doesn't happen in the original game and I have already restarted my phone once and re-installed canvas too, please help I really need candles for the new season :( Is it because of the new update? It has been like this for at least 8 hours now. Thanks beforehand.

**Reply by Krisstobal** [reply, marked as answer]
It seems that one of the mods was the one that made the mod stop working, i forgot to answer this, im sorry.
